shared/fixes/bash/accounts_authorized_local_users_sidadm_orasid.sh

```diff
@@ -8,6 +8,21 @@ default_os_user="root"
 # add users sidamd, orasid, sapadm and oracle if needed
 userlist="root"
 sapmnt_SID_stem="/sapmnt/[A-Z][A-Z0-9][A-Z0-9]"
+oracle_SID_stem="/oracle/[A-Z][A-Z0-9][A-Z0-9]"
+
+# if owner of any directory or file in the given list is the user oracle, return 1.
+# otherwise return 0. 
+# Usage: is_owner_oracle "${list[@]}"
+function is_owner_oracle {
+	local path_list=("$@")
+	local is_oracle=0
```

Comment:
You use 0 and 1, which may be slightly confusing (e.g. commands `true` returns 0 and `false` returns 1). I incline to use "yes" and "no" instead of numbers.

Reply:
Good suggestion, I will modify 0 and 1 to no and yes. Thanks.